instrumentation/aws_lambda/lib/opentelemetry/instrumentation/aws_lambda/handler.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module AwsLambda
+      # Handler class that create span around the _HANDLER
+      class Handler
+        attr_reader :handler_method, :handler_class
+
+        def initialize
+          @flush_timeout = ENV.fetch('OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT', '30000').to_i
+        end
+
+        # Extract context from request headers
+        def call_wrapped(event:, context:)
+          parent_context   = extract_parent_context(event)
+          span_attributes  = event['version'] == '2.0' ? v2_proxy_attributes(event) : v1_proxy_attributes(event)
+          original_handler = resolve_original_handler
+          response         = call_original_handler(event: event, context: context)
+
+          span_attributes[OpenTelemetry::SemanticConventions::Trace::HTTP_STATUS_CODE] = response['statusCode'] if response.instance_of?(Hash) && response['statusCode']
+          span_attributes[OpenTelemetry::SemanticConventions::Resource::FAAS_ID]       = context.invoked_function_arn
+          span_attributes[OpenTelemetry::SemanticConventions::Trace::FAAS_EXECUTION]   = context.aws_request_id
```

Comment:
It's nice to see these attributes. Is there a spec you followed to add these specific items? I found this: https://opentelemetry.io/docs/specs/semconv/faas/

Reply:
Not really following the spec. This code is copied from splunk and also took reference on python lambda.

Reply:
Maybe we should follow-up this PR with a few issues to bring the instrumentation into compliance? I think the key specs are: 
* [faas spans](https://opentelemetry.io/docs/specs/semconv/faas/faas-spans/)
* [aws lambda](https://opentelemetry.io/docs/specs/semconv/faas/aws-lambda/)